notes/2024-02-06 biglep feedback for PR #65.md

```diff
@@ -0,0 +1,99 @@
+# Purpose
+This is intended to accompany https://github.com/ipld/github-mgmt/pull/65.
+I did it as its own file to enable threaded replies on specific lines/sections of this file with the PR UI.
+If I didn't do this, we'd be limited to quote replies in https://github.com/ipld/github-mgmt/pull/65.
+Given that I expect a decent amount of back-and-forth on some of these suggestions, I figured quote replies would get unweildy.
+I'm not assuming this file will get checked in.  It's purely to help with the code review process.
+Maybe at the end if there is useful content here, we copy/paste it into the PR itself.  (Alternatively we could merge it in.)
+Or I guess we could just move into Github discussions?  (I didn't do that because wanted to keep all the context here.)
+Obivously I could have done this in another tool like Google Docs, but I wanted to keep the discussion more open and discoverable (especially since this will the basis for changes to libp2p, ipfs, etc. repos).
+If I should do something else, please let me know!
+
+# General github-mgmt feedback/wishes
+## Documentation I wish was more readily accessible when using github-mgmt
+Below are links or lookups I wish more readily available when engaging with github-mgmt.  
+I find myself looking them up each time I'm thinking about github-mgmt.  
+I didn't see them in https://github.com/ipld/github-mgmt/tree/master/docs.  
+
+* Org-level permissions: https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#permissions-for-organization-roles
+* Org base permissions: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/setting-base-permissions-for-an-organization#setting-base-permissions
+* Permissions for each repository role: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role 
+* Difference between team maintainer vs. member: https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member
+
+Reminders I wish were more present:
+* Terraform “push” corresponds with Github “write”.
+* Terraform “pull” corresponds with Github “read”.
+* Terraform members.admin correspond with Github "org owner" per https://registry.terraform.io/providers/integrations/github/latest/docs/resources/membership
+
+## User-oriented view of changes
```

Comment:
2024-02-08 verbal conversation: it was agreed that this is needed before doing a large permissions cleanup campaign.  @galargh is going to take this.  @BigLep will create a general github-mgmt backlog item for it.  @galargh expects to have something here by EOD 2024-02-14.